cotk/metric/metric.py

```diff
@@ -63,9 +63,7 @@ def forward(self, data):
 			data (dict): A dict at least contains the following keys.
 			data[reference_allvocabs_key] (list of list of list): Reference sentences.
 				Does not contain start token (eg: ``<go>``) and end token (eg: ``<eos>``).
-				Outermost list: batch_size
-				Innermost list: number of words, allow different sizes
-				Second innermost list: number of sentences, allow different sizes
+				Size: `[batch_size, sentence_num, max_sentence_length]`
```

Comment:
It's important to claim that the 2nd and 3rd dimensions can be different sizes. It's different from a matrix with fixed length

Reply:
Maybe you can add a example for that. Find tensorflow or pytorch for reference

Reply:
got it